src/Client/BuzzHouse/AST/SQLProtoStr.cpp

```diff
@@ -2689,9 +2757,20 @@ CONV_FN(CreateTableAs, create_table)
 
 CONV_FN(CreateTable, create_table)
 {
-    ret += create_table.replace() ? "REPLACE" : "CREATE";
+    switch (create_table.create_opt())
+    {
+        case CreateTable_CreateTableOption::CreateTable_CreateTableOption_Create:
```

Comment:
Why are these names so incredible ugly? Why do we need that many `Create`s? I don't understand why this is not an enum with `CreateTableOption` having different values: `Create`. `Replace`, etc...

Reply:
It is an enum generated by Protobuf. I am not sure if I can change this.